### DISCUSSION TITLE
UNABLE TO RENEW MY DATACAMP SUBSCRIPTION

### DISCUSSION THREAD

**Discussion by Ajohncode** [community/community#110540] [GitHub Education]
### Hi everyone,

Hi everyone, 
I'm Ajohn. I applied for GitHub student Developer pack around 2022 and I was given all the offers (Including DATACAMP). I really enjoyed the 3-month Datcamp subscription because it enabled me to learn DS/ML (Not yet done though). 

Recently, I mostly needed DATACAMP subscription to complete my data science courses. I tried applying to GitHub student developer pack again for renew, I wasn't given. I want to really complete some courses but I'm unable to cos i can't afford the fee.
 
Please is Datacamp subscription on GitHub Student Developers Pack ONCE??? and Please any suggestion on how to get the Subscription. I need to finish my courses (ASAP). 

I will be eager to hear from anyone.
Thanks.

***NOTE: I'M STILL A STUDENT. HAVE DONE A RENEW APPLICATION BUT DATACAMP SUBSCRIPTION WAS NOT GIVEN (Was given others tho). Lo,I need this BADLY.

More

**Comment by hayahahmed20** [comment]
did you find any solution?

**Comment by Ajohncode** [comment]
Noo!!!

The communities are not yet given any subscription out yet

Still stranded want to take and finish my courses badly    : (
